### ISSUE TITLE
Not the maximum resolution of the camera.

### ISSUE THREAD

**Issue by yuya-tani** [purple-technology/react-camera-pro#33]
Hi.

In the Features part, there is "with FullHD resolution (or maximum supported by camera)."
How can I get the maximum resolution of the camera?

The camera supports 4K size, but even if aspectRatio is set to "16/9",
When you do "takePhoto", it is a FullHD (1980 x 1080) photo.

Is there any other way to set it?

Best regard.

**Comment by PercheEtienne** [comment]
Hi,

I have the same issue, did you find a solution ? 

Thanks

**Comment by RyoheiTomiyama** [comment]
Maybe, max resolution is 1980x1980.
https://github.com/purple-technology/react-camera-pro/blob/master/src/components/Camera/Camera.tsx#L168-L176

For cameras with lower performance, it will be the camera's best image quality.

**Comment by nicholas-aponte** [comment]
> Maybe, max resolution is 1980x1980. https://github.com/purple-technology/react-camera-pro/blob/master/src/components/Camera/Camera.tsx#L168-L176
> 
> For cameras with lower performance, it will be the camera's best image quality.

Is there a technical reason for this? or could the resolution be set via props so we can increase it?

**Comment by RyoheiTomiyama** [comment]
> Is there a technical reason for this?

No, I used [react-webcam](https://www.npmjs.com/package/react-webcam) and it worked!
https://codesandbox.io/p/github/RyoheiTomiyama/react_webcam_feature/main?file=/src/Camera/Camera.tsx:1,1

> could the resolution be set via props so we can increase it?
